Cargo.toml

```diff
@@ -48,8 +48,9 @@ repository = "killercup/cargo-edit"
 repository = "killercup/cargo-edit"
 
 [dependencies]
+cargo_metadata = "0.3.0"
 docopt = "0.8"
-error-chain = "0.10.0"
+error-chain = "0.11.0"
```

Comment:
Oh nice, didn't see this was stable already! It impls ResultExt for Options, ~~maybe we can use that in some places?~~ and I see you're using that, awesome!